cli/src/cmd/forge/script.rs

```diff
@@ -89,8 +111,10 @@ impl Cmd for ScriptArgs {
         let verbosity = evm_opts.verbosity;
         let config = Config::from_provider(figment).sanitized();
 
-        let mut nonce = if let Some(ref fork_url) = evm_opts.fork_url {
-            foundry_utils::next_nonce(evm_opts.sender, fork_url, None)?
+        let fork_url =
+            evm_opts.fork_url.as_ref().expect("You must provide an RPC URL (see --fork-url).");
```

Comment:
If I recall, it's about the `nonce`. Having it default to 0 because there's no RPC, just does not feel right. Especially, since contract addresses depend on it, and in the end, feels like an accident waiting to happen.

I'd rather have `forge test` or `forge run` to actually test stuff out regardless of nonces.